doc/deimos.rst

```diff
@@ -29,17 +29,7 @@ Deviations
 
 Here are the main deviations from the default settings
 for DEIMOS
-(see :func:`~pypeit.spectrographs.keck_deimos.KeckDEIMOSSpectrograph.default_pypeit_par`)::
-
-
-    # Default lamps
-    par['calibrations']['wavelengths']['lamps'] = ['ArI','NeI','KrI','XeI']
-    # Do not require bias frames
-    turn_off = dict(use_biasimage=False)
-    par.reset_all_processimages_par(**turn_off)
-    # Spectral flexure correction
-    par['flexure']['spec_method'] = 'boxcar'
-
+(see :func:`~pypeit.spectrographs.keck_deimos.KeckDEIMOSSpectrograph.default_pypeit_par`).
```

Comment:
I would edit this to:
```
The default changes to the ``PypeIt`` parameters specific to DEIMOS 
data are listed here: :ref:`instr_par`.
```
We could also edit the script that generates `pypeit_par.rst` to provide reference links so that we could point to each instrument individually.

Reply:
edited